res/values/strings.xml

```diff
@@ -859,12 +876,15 @@
     <string name="notify_dismiss">Dismiss</string>
     <string name="notify_reply_button">Reply</string>
     <string name="notify_new_message">New message</string>
+    <string name="notify_new_messages">New messages</string>
```

Comment:
just a hint, no blocker: there is also `chat_n_new_messages` - this is maybe even nicer?